docs/cookbook/deployment/introduction-to-sylius-cloud.rst

```diff
@@ -0,0 +1,100 @@
+- \- **Deployment Process** - the process of replacing the existing application with the already built, updated version. The current application version is hosted until the whole deployment process exits without an error.
+- \- **Relations** - the connections established between different services or components within your project. These connections enable communication and interaction between services, allowing them to work together seamlessly to support your application.
+
+How to deploy Sylius to Sylius Cloud by Platform.sh?
+====================================================
+
+.. tip::
+
+    Start with reading `Platform.sh documentation <https://docs.platform.sh/frameworks/symfony.html>`_.
+    Also Symfony provides `a guide on deploying projects to Platform.sh <https://symfony.com/doc/current/deployment/platformsh.html>`_.
+
+The process of deploying projects to Sylius Cloud by Platform.sh is based on the guidelines prepared for Symfony projects in general.
+In this guide you will find sufficient instructions to have your application up and running.
+
+0. Project configuration
+------------------------
+
+In order for your project to be ready for Sylius Cloud, make sure to have it initiated using this command:
+
+.. code-block:: bash
+
+    composer create-project sylius/sylius-standard acme
+
+This way you'll have all the infrastructure configuration files from `Sylius-Standard <https://github.com/Sylius/Sylius-Standard>`_.application
+
+The files you should be interested in are:
+
+- \- `.platform.app.yaml` - This file is used to define the configuration of the application itself. It specifies how the application should be built, what runtime to use, which commands to run on deployment, etc. It also includes information such as the type of application, language runtime, build and deploy hooks, and more.
+- \- `.env.platform.dist` - The environment file automatically used by the application during the build process.
+- \- `.platform/routes.yaml` - This file defines the routes and how traffic is routed to the application. It specifies the incoming URLs and how they should be directed to the appropriate services or applications. Routes can be configured to handle different paths, domains, or other conditions.
+- \- `.platform/services.yaml` - This file is used to define additional services or dependencies required by the application. It allows defining various types of services such as databases, caches, search engines, etc., along with their configuration. Services defined in this file are automatically provisioned and integrated with the application environment.
+
+1. Prepare a Platform.sh project
+--------------------------------
+
+* Create an account on `Platform.sh <https://platform.sh/>`_.
+
+* Create a new project, name it (**MyFirstShop** for example) and select the **Blank project** template.
```

Comment:
We also need to create an organisation, then we will be asked to provide phone number for the sms code to authorize, if we haven't done it earlier, for someone it could be blocking, I would be after mentioning it before creating project like "if it's your first time with platform.sh you may be asked to create your organisation, do it, then you will be asked to provide phone number, for the sms code authorization, don't worry it is not stored, it is just for the security reasons" or something like that